sass/components/_page.scss

```diff
@@ -0,0 +1,22 @@
+$font-family: 'Roboto', 'sans-serif';
+
+.page {
+  font-family: $font-family;
+  &__body {
+    margin: 0;
+  }
+  &__body--no-scroll {
```

Comment:
here you could use nesting, as modifier `&--no-scroll` directly related to `&__body`